public/locales/en/translation.json

```diff
@@ -19,6 +19,12 @@
     }
   },
   "home": {
+    "genderViolenceMain": {
+      "headerPartOne": "You are not alone Safe Place is",
```

Comment:
i think you're a bit confused about the principles here
* There were previously two parts separated by a linebreak
" You are not alone"
"Safe place is XXXX"
"XXXX" = here (in english). burada, in turkish, whatever it is in arabic.
* it should support any language in any order
* the idea of this sentence, if we get really abstract, is "statement statement {{emphasized thing}}"
* think of it as a function
```
herePlaceholder => "Safe Place is ${here}"
```

so now in arabic, let's say it's in a different order. then the header is like so:
genderViolenceMain.aloneStatement: "FOO FOO FOO" (You are not alone, in arabic)
genderViolenceMain.weAreHere: "{{here}} BAR BAR BAR" (We are X, in arabic)
genderViolenceMain.here: "BAZ" (Here, in arabic)

so now, when you do weAreHere, you can use dangerouslySetInnerHtml (again tlak to Jennet) do make it so that .weAreHere can take a <span>{t.genderViolenceMain.here}</span> and you end up avoiding all code in the translation

Reply:
^ notice how the thought process at the end is the language, abstractly. i don't speak arabic, i don't need to. my interpretation of arabic is apparently just foo bar baz. 

but i just need to know that if i wanted to support 50 languages, i shouldn't be assuming anything about the order of the languages; all i have is three statements, one of which represents the emphasized thing (which DOESN'T have to be at the end of the sentence. it can be anywhere, because i used `{here}` to interpolate)>

Reply:
But if I add the {{here}} in translation in the dangerouslySetInnerHtml it'll have to be like this:

 dangerouslySetInnerHTML={{
                __html: t("home.genderViolenceMain.headerPartTwo", {
                  here: "<span>HERE</span>",
                }),
              }}

which means that the emphasized word will not be translated into Arabic and it will like the attached screenshot, even in the chatbot where jennet used dangerouslySetInnerHTML it looks like the number is fixed in <a> tag. I don't know if I am still missing something 😕 

![Screen Shot 2020-11-20 at 11 53 23 PM](https://user-images.githubusercontent.com/58190902/99849522-c70f0c00-2b8c-11eb-8484-889f65258af3.png)

Reply:
I am sorry for giving you trouble over a simple topic like this.

Reply:
In general not related to this specific issue, the problem in translation from english/arabic is related more to writing from right to left rather than order of sentence. 
For example, the name of the website is not translated because it's kinda the identity of the website. 
So when we have a sentence that is part from rtl and part from ltr it becomes more tricky; and parts of the text becomes shifted sometimes.